Tools/parametric_model/generate_parametric_model.py

```diff
@@ -104,6 +105,8 @@ def start_model_estimation(
         model.load_dataframes(
             select_visual_data(model.data_df, visual_dataframe_selector_config_dict)
         )
+        model.prepare_regression_matrices()
+        model.compute_fisher_information()
```

Comment:
@Jaeyoung-Lim Even in case you should decide to keep the first regression matrix and fisher information matrix computations where they were before, these lines look like some that went missing during some earlier changes 😅

Reply:
Right, maybe this was why it got faster when we configured it for the setpoint based data selection?

Reply:
Yes, I think that was the reason for the improved runtime :)